1.1.c

```diff
@@ -0,0 +1,34 @@
+#include<stdio.h>
+#include <conio.h>
+
+#define N 10
+
+void test()

```

Comment:
```suggestion
bool isArrayWithoutDuplicates(int *array, int size)
```
1) Абсолютно неговорящее название функции.
2) Функция должна проверять, есть ли дубликаты в массиве и возвращать ответ.
3) Функция должна принимать массив и его размер в качестве аргумента.
(-1 балл)